tests/cli/test_rasa_data.py

```diff
@@ -1,6 +1,9 @@
 import os
+import pytest
+from collections import namedtuple
 from typing import Callable
 from _pytest.pytester import RunResult
+from rasa.cli.data import validate_files
```

Comment:
```suggestion
from rasa.cli import data
```
We try to import modules rather than methods as much as possible, to make it clear where methods are coming from

Reply:
> Functionality looks great  Just some style comments and a revision to the docs after what we changed last time and we're good.
> 
> P.S. sorry to have you waiting and re-merging master, I wasn't aware you were ready (didn't get a notification), in the future feel free to just drop a comment when all the mentioned changes have been integrated and it's ready for another look!

Ok, no problem : )